versions.js

```diff
@@ -0,0 +1,16 @@
+/*
+Site-wide variables, usage:
+- include in MDX/JSX (has to be in a tag)
+- doesn't work with pure Markdown
+- import the file first: import {siteVariables} from '@site/versions';
+- then reference like: <tag>{siteVariables.variableName}</tag>
+ */
+export const siteVariables = {
+    javaModuleParentVersion: 'parentversion',
+    javaCommonsVersion: 'commonsversion',
+    javaApiVersion: 'apiversion',
+    javaMessagingVersion: 'messagingversion',
+    javaCqrsVersion: 'cqrsversion',
+    javaAzureCosmosVersion: 'cosmosversion',
+    javaAzureServicebusVersion: 'servicebusversion'
```

Comment:
@johnsall / @guillermogutierrez : do you mind providing the current list of the used modules? These are injected into the individual modules' `pom.xml` `dependency` version sections.

Reply:
edit: currently all are set to `1.0.0`.